mne/channels/channels.py

```diff
@@ -439,13 +439,13 @@ class UpdateChannelsMixin(object):
     def pick_types(self, meg=True, eeg=False, stim=False, eog=False,
                    ecg=False, emg=False, ref_meg='auto', misc=False,
                    resp=False, chpi=False, exci=False, ias=False, syst=False,
-                   seeg=False, bio=False, ecog=False, include=[],
-                   exclude='bads', selection=None):
+                   seeg=False, dipole=False, gof=False, bio=False, ecog=False,
+                   include=[], exclude='bads', selection=None):
```

Comment:
I saw it as a bug. There are more plotting functions for which I've messed with the parameter list to make them consistent. A good example of this is `mne.viz.plot_ica_properties` where I flipped the first two parameters to make it consistent. For that one I figured since the function isn't part of any stable release yet I could get away with it :)